english/cry/cry.md

```diff
@@ -0,0 +1,3 @@
+As you cry and cry, your tears fill the cup you're holding and soon you have a cup full of salty tears. And you can't do much with that!
+
+[Why don't you just call your mom?](mom/mom.md)
```

Comment:
This link would work if you moved the `mom` file and directory inside of the `cry` directory.